Competition2017/src/competition/subsystems/autonomous/selection/SetupBreakBaselineCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.autonomous.selection;
+
+import competition.subsystems.drive.commands.DriveForDistanceCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SetupBreakBaselineCommand extends BaseAutonomousCommandSetter {
+
+    public final DriveForDistanceCommand breakBaselineAuto;
+    // distance to break baseline from a position with both back wheels on the wall
+    private DoubleProperty distanceFromWallToBaseline;
+
+    public SetupBreakBaselineCommand(AutonomousCommandSelector autonomousCommandSelector,
+            DriveForDistanceCommand driveForDistance, XPropertyManager propMan) {
+        super(autonomousCommandSelector);
+        distanceFromWallToBaseline = propMan.createEphemeralProperty("distance from wall to break baseline in inches", 96);
+        this.breakBaselineAuto = driveForDistance;
```

Comment:
my only thought here is that the name of the auto command will be just "DriveForDistanceCommand" but if we only have the 1 auto program using that it might not be so bad